specificatie/openapi.yaml

```diff
@@ -648,6 +810,132 @@ components:
               items:
                 $ref: '#/components/schemas/ZoekResultaatHal'
 
+    GeoJsonZoekOpdracht:
+      description: 'Geometrie waarmee gezocht moet worden naar resources.'
+      type: object
+      required:
+        - geometrie
+      properties:      
+        geometrie:
+          $ref: '#/components/schemas/Geometrie'
+        spatialOperator:
+          type: string
+          description: >-
+            "Een operator uit de set van spatial operators die gedefinieerd zijn in ISO 19125-1. Ondersteund worden:
+            - intersects  : object geometrie `overlapt` of `raakt` de opgegeven contour
+            - within      : object geometrie valt volledig binnen de opgegeven contour
+            - contains    : object geometrie omsluit de opgegeven contour volledig"
+          enum:
+            - intersects
+            - within
+            - contains
+          default: intersects
+        tolerantie:
+          description: 'De maximale afwijking (in meters) tussen twee punten waarbij deze punten nog als één punt worden gezien.'
+          type: number
+          format: float
+          default: 0.001
+          minimum: 0.00001
+          maximum: 1
+
+    Geometrie:
+      type: object
+      description: 'De geometrie waarmee gezocht moet worden naar objecten. Er kan een punt, cirkel of polygoon worden opgegeven.' 
+      properties:
+        locatie:
```

Comment:
Het gaat hier om een punt, dus zoekparameter "punt" noemen is denk ik duidelijker dan "locatie" om het onderscheid met cirkel of vlak aan te geven. In de response noemen we dit ook "punt" (in PuntOfVlak)
@CathyDingemanse @MelvLee @JohanBoer bij panden hebben we ook de query parameter "locatie" genoemd, niet "punt". Vinden jullie het beter om hierbij aan te sluiten (zoals @strijm hier heeft gedaan) of om het "punt" te noemen?